.github/workflows/artifacts.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   upload-s3:
     name: cd/Upload artifacts to S3
     runs-on: ubuntu-22.04
-    if: github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'
+    if: github.repository_owner == 'mattermost' && github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'
```

Comment:
You need to also add this to the next step for the docker build because the artifact is downloaded after uploaded

Reply:
Are you sure about that? If `upload-s3` doesn't run, I don't think it would ever try to run `build-docker`.

I could still change it for consistency though

Reply:
I added it just to be safe